application/src/main/java/org/wecancodeit/serverside/model/Comment.java

```diff
@@ -1,4 +1,33 @@
 package org.wecancodeit.serverside.model;
 
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.Id;
+import javax.persistence.ManyToMany;
+import java.util.Collection;
+@Entity
 public class Comment {
+
+    @Id
+    @GeneratedValue
+    private String commentName;
```

Comment:
We probably want the generatedValue to be an id number, especially since commentName is included in the constructor